manifest.json

```diff
@@ -15,6 +15,11 @@
         "scripting",
         "storage"
     ],
+    "registerScripts": [{
+        "run_at": "document_idle",
+        "js": ['content-script.js'],
+        "matches": ['*://*.youtube.com/*']
```

Comment:
Not a requirement in json files, but the standard I believe is to just use double quotes **_everywhere_** which would make it easier to comply with the spec so it's something you don't ever need to worry about.